src/utils/validateAndParseAddress.ts

```diff
@@ -11,3 +11,18 @@ export function validateAndParseAddress(address: string): string {
     throw new Error(`${address} is not a valid address.`)
   }
 }
+
+// Checks a string starts with 0x, is 42 characters long and contains only hex characters after 0x
+const startsWith0xLen42HexRegex = /^(0x)?[0-9a-fA-F]{40}$/
```

Comment:
probably should remove the `?` and group `()`  to enforce the `0x` prefix?

Reply:
Oops, great catch!